pymc3/stats.py

```diff
@@ -509,6 +509,9 @@ def df_summary(trace, varnames=None, stat_funcs=None, extend=False, include_tran
         else:
             varnames = [name for name in trace.varnames if not name.endswith('_')]
 
+    if batches is None:
+        batches = min([100, len(trace)])
```

Comment:
I think trace is a dictionary here -- could this happen down after `flat_vals` is defined?